region/crateria/east/East Ocean.json

```diff
@@ -159,25 +159,63 @@
         }
       }
     },
+    {
+      "link": [1, 1],
+      "name": "Leave With Mockball (Space Jump)",
+      "requires": [
+        "canPreciseSpaceJump"
+      ],
+      "exitCondition": {
+        "leaveWithMockball": {
+          "remoteRunway": {
+            "length": 18,
+            "openEnd": 1,
+            "steepUpTiles": 1,
+            "steepDownTiles": 2
+          },
+          "landingRunway": {
+            "length": 8,
+            "openEnd": 1,
+            "steepUpTiles": 1
+          }
+        }
+      }
+    },
     {
       "link": [1, 1],
       "name": "Leave With Temporary Blue",
       "requires": [
         "Gravity",
-        {"canShineCharge": {
-          "usedTiles": 20,
-          "openEnd": 0,
-          "steepUpTiles": 4,
-          "steepDownTiles": 2,
-          "startingDownTiles": 1
-        }},
         "canLongChainTemporaryBlue",
-        "canXRayTurnaround",
-        "can4HighMidAirMorph"
+        "can4HighMidAirMorph",
+        {"or": [
+          {"and": [
+            {"canShineCharge": {
+              "usedTiles": 20,
+              "openEnd": 0,
+              "steepUpTiles": 4,
+              "steepDownTiles": 2,
+              "startingDownTiles": 1
+            }},
+            "canXRayTurnaround"  
+          ]},
+          {"and": [
+            {"canShineCharge": {
+              "usedTiles": 22,
+              "openEnd": 0,
+              "steepUpTiles": 3,
+              "steepDownTiles": 3
+            }},
+            "HiJump"
```

Comment:
Not sure where HiJump is supposed to be used.  You can gravity jump the water sections, and barely jump morph to reach the above water platforms.

Turning gravity off helped with the underwater jump morphs, but the ones by the door are still hard.

You can go under the structure facing left and not need any xray turnarounds either.

Reply:
I was using HiJump to get onto the first ocean platform. Wasn't thinking of Gravity jumping. Good catch. Added "canGravityJump" option and added a little more detail to the note. Added a devNote for the tight morph option.

Reply:
Oh, took me a minute to realize what you meant, that you can also use the left runway right-to-left without X-Ray. Knowing that, I don't think there would be any reason to use the right runway, unless Gravity jumps were not possible for some reason. Based on that, simplified the strat & note more now.